packages/pouchdb-ajax/src/request-browser.js

```diff
@@ -117,6 +117,33 @@ function xhRequest(options, callback) {
     xhr.abort();
   };
 
+  var ret = {abort: abortReq};
+
+  var doSetTimeout = function () {
+    if (timer !== undefined) {
```

Comment:
I dont think we need these `undefined` checks, we can unconditionally clear and set the timeout, `clearTimeout(undefined)` doesnt do anything bad